Make terms configurable (#3)
bbugh/action-fixme-check: README.md

# action-fixme-check

> Checks the code base for any case-insensitive `FIXME:` or `WIP:` (with the colon), 
> you can configure the term using `terms`, and fails the check if any are found. 

Reviewer: ```suggestion
Checks the code base for any terms ending with a colon, and fail the check if
any are found. The default term is `FIXME:`. You can add or change the terms
using the `term` parameter, see [Installation](#Installation) below.

```

    steps:
    - uses: actions/checkout@v1
    - uses: bbugh/action-fixme-check@master # or @ the latest release
      with:
      	terms: 'wip|fixme'
```

## Support

- [Official workflow configuration docs](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions)

## Testing words

- I am using `WIP:` here.
- I am using `FIXME:` here.
- Nothing to see here.
- I am using `FIX:` here.
- I am using `wip:` here.
- I am using `fixme:` here.

> 

Reviewer 2: What's the purpose of this addition?

    steps:
    - uses: actions/checkout@v1
    - uses: bbugh/action-fixme-check@master # or @ the latest release
      with:
      	terms: 'WIP|FIXME' # optional, defaults to `FIXME`
```

## Support

- [Official workflow configuration docs](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions)

## Testing words

(Used for testing this action on itself.)

- I am using `WIP:` here.
- I am using `FIXME:` here.
- Nothing to see here.
- I am using `FIX:` here.

> - I am using `wip:` here.
> - I am using `fixme:` here.

Reviewer: ```suggestion
```

    steps:
    - uses: actions/checkout@v1
    - uses: bbugh/action-fixme-check@master # or @ the latest release
      with:
      	terms: 'WIP|FIXME' # optional, defaults to `FIXME`

>       	case-sensitive: false  # optional, defaults to `false`

Reviewer: ```suggestion
```
